CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 ## 4.67.0 (Unreleased)
+FEATURES:
+* **New Data Source:** `google_**_iam_policy` ([#14662](https://github.com/hashicorp/terraform-provider-google/pull/14662))
+* **New Data Source:** google_vertex_ai_index ([#14640](https://github.com/hashicorp/terraform-provider-google/pull/14640))
+
+IMPROVEMENTS:
+* cloudrun: Added beta field `template.spec.columes.empty_dir` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
+* cloudrun: Added field `template.spec.containers.name` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
+* compute: - promoted field `network_performance_config` to ga ([#14678](https://github.com/hashicorp/terraform-provider-google/pull/14678))
+* compute: added `guest_os_features` and `licenses` fields to `google_compute_disk` and `google_compute_region_disk` ([#14660](https://github.com/hashicorp/terraform-provider-google/pull/14660))
+* datastream: added `mysql_source_config.max_concurrent_backfill_tasks` field to `google_datastream_stream` resource ([#14639](https://github.com/hashicorp/terraform-provider-google/pull/14639))
+* firebase: added additional import formats for app resources, removed the need to supply duplicate projects in some cases ([#14638](https://github.com/hashicorp/terraform-provider-google/pull/14638))
```

Comment:
this should possibly be rewritten to emphasize user impact / not use as much technical language